000-vip-init.php

```diff
@@ -248,6 +248,14 @@
 	$headers['X-Powered-By'] = 'WordPress VIP <https://wpvip.com>';
 	$headers['Host-Header']  = 'a9130478a60e5f9135f765b23f26593b'; // md5 -s wpvip
 
+	// By default, the 'vip_index_noprod_domains' filter returns an empty array (to prevent indexing any non-production sites).
+	$index_noprod_domains = apply_filters( 'vip_index_noprod_domains', '__return_empty_array' );
+
+	// Skip "noindex" and "nofollow" the site if the filter 'vip_index_noprod_domains' returns true or contains the current domain.
+	if ( true === $index_noprod_domains || in_array( $_SERVER['SERVER_NAME'], $index_noprod_domains, true ) ) {
```

Comment:
This check is problematic. Assuming we fix the above note, since we're expecting an array this is what we should check for. Something like ```is_array( $index_noprod_domains ) && in_array....`